Specs/Scene/ModelSpec.js

```diff
@@ -509,6 +510,57 @@ defineSuite([
         });
     });
 
+    it('renders a model with the KHR_binary_glTF extension', function() {
```

Comment:
I'm not sure what you want here. There is only 1 binary test model and we have it in `bgltf` and `glb` formats.